app/views/registrations/register.html.erb

```diff
@@ -10,7 +10,7 @@
   <% opens_in_days = distance_of_time_in_words_to_now(@competition.registration_open) %>
   <% closes_in_days = distance_of_time_in_words_to_now(@competition.registration_close) %>
 
-  <% if @competition.registration_not_yet_opened? %>
+  <% if !@competition.uses_new_registration_service? && @competition.registration_not_yet_opened? %>
```

Comment:
I suspect this is womehow linked to the above change in `should_render_register_v2?`

Reply:
yes